identity/client/src/utility/localization/en/groups.json

```diff
@@ -0,0 +1,27 @@
+{
+  "edit": "Edit",
+  "delete": "Delete",
+  "groups": "Groups",
```

Comment:
Those translations are not need since they exists in `components.json` and we fallback to `components.json` in case we do not find a translation here.